runtime/src/bank.rs

```diff
@@ -3257,6 +3281,49 @@ impl Bank {
         self.set_compute_budget(compute_budget);
     }
 
+    fn reconfigure_token2_native_mint(self: &mut Bank) {
+        let reconfigure_token2_native_mint = match self.operating_mode() {
+            OperatingMode::Development => true,
+            OperatingMode::Preview => self.epoch() == 95,
+            OperatingMode::Stable => self.epoch() == 75,
+        };
+
+        if reconfigure_token2_native_mint {
+            let mut native_mint_account = solana_sdk::account::Account {
+                owner: inline_spl_token_v2_0::id(),
+                data: inline_spl_token_v2_0::native_mint::ACCOUNT_DATA.to_vec(),
+                lamports: 1000000000, // 1 SOL
```

Comment:
nits: `1_000_000_000`

Reply:
I'm disappointed that clippy didn't yell at me here.  Thanks

Reply:
Could you avoid the big number and counting zeroes altogether with https://github.com/solana-labs/solana/blob/3db246f596ab7e5f220adfe9a6961cf31e59c71c/sdk/src/native_token.rs#L10

Reply:
Oh fine, I'll `use` it :)